src/algebra/Mon/colimits.lean

```diff
@@ -1,3 +1,6 @@
+-- Copyright (c) 2019 Scott Morrison. All rights reserved.
```

Comment:
Style guide calls for block comment here. (It's good to keep this uniform. The dumb script I wrote to identify files without headers got tripped up on a bunch like this.)

Reply:
Done.